src/mono/wasm/runtime/http.ts

```diff
@@ -16,6 +17,22 @@ function verifyEnvironment() {
     }
 }
 
+export function http_wasm_supports_streaming_request(): boolean {
+    if (typeof Request !== "undefined" && "body" in Request.prototype && typeof ReadableStream === "function") {
+        let duplexAccessed = false;
+        const hasContentType = new Request("", {
```

Comment:
Could we add here a comment explaining the logic behind feature detection? Something similar to what is in the referenced article.

Reply:
Done.